### PR TITLE
Changes in DartVmServiceDebugProcess updating getUrisForFile and getSourcePosition to support Bazel workflows

### PR DESCRIPTION
**Pull request by jwren**
@alexander-doroshko

### CODE REVIEW

**Comment by jwren** [comment]
When you are pushing commits to old IntelliJ versions, please push back to `2019.2.3`:

```
IntelliJ IDEA 2019.2.3 (Ultimate Edition)
Build #IU-192.6817.14, built on September 23, 2019
```

File: Dart/src/com/jetbrains/lang/dart/util/DartBazelFileUtil.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  private static final String WORKSPACE_FILE_NAME = "WORKSPACE";
+  private static final String READONLY_FOLDER_NAME = "READONLY";
+
+  @Nullable
+  public static VirtualFile getBazelWorkspace(@NotNull final VirtualFile file) {
+    VirtualFile parent = file.getParent();
+    while (parent != null) {
+      final VirtualFile readonlyFolderVFile = parent.findChild(READONLY_FOLDER_NAME);
+      if (readonlyFolderVFile != null && readonlyFolderVFile.exists()) {
```
Comment (alexander-doroshko) [inline Dart/src/com/jetbrains/lang/dart/util/DartBazelFileUtil.java]: `findChild()` can't return a file that doesn't exist, so I removed additional `exists()` check.
You may add `isDirectory()` check if you want.

**Comment by alexander-doroshko** [comment]
Pushed. Expect a slight delay with guthub mirroring.

> When you are pushing commits to old IntelliJ versions, please push back to 2019.2.3

I usually cherry-pick to branch 192 and when publishing I set 'since-build' to 192.5728, so it's compatible with all 2019.2.x releases.

**Comment by jwren** [comment]
Thanks Alex

File: Dart/src/com/jetbrains/lang/dart/util/DartBazelFileUtil.java (hunk 1 above)
Comment (jwren) [reply Dart/src/com/jetbrains/lang/dart/util/DartBazelFileUtil.java]: Thanks Alex, I'll go add the `isDirectory()` as soon as the github mirroring catches up.

**Pull request closed without merging**